raiden/tests/utils/events.py

```diff
@@ -125,34 +123,3 @@ def must_have_events(event_list: List[TM], *args) -> bool:
             return False
 
     return True
-
-
-def wait_for_raiden_event(
-    raiden: RaidenService, item_type: Type[Event], attributes: Mapping, retry_timeout: float
-) -> Optional[Event]:
-    """Wait until an event is seen in the WAL events
-
-    Note:
-        This does not time out, use gevent.Timeout.
-    """
-    found = None
-    while found is None:
-        found = raiden_events_search_for_item(raiden, item_type, attributes)
-        gevent.sleep(retry_timeout)
```

Comment:
Isn't there a signaling mechanism that wakes up this thread, whenever the desired event is seen?  Or is polling the only option?